app/src/ui/branches/pull-request-status.ts

```diff
@@ -0,0 +1,29 @@
+import { PullRequestStatus } from '../../models/pull-request'
+import { APIRefState } from '../../lib/api'
+import { assertNever } from '../../lib/fatal-error'
+
+export function toFriendlyText(state: APIRefState): string {
+  switch (state) {
+    case 'failure':
+      return 'failed'
+    case 'pending':
+    case 'success':
+      return state
+
+    default:
+      return assertNever(state, `Unknown APIRefState value: ${state}`)
+  }
+}
+
+export function getSummary(prStatus: PullRequestStatus): string {
```

Comment:
Could we document this please? Additionally, could we name it `getPRStatusSummary` or something similar?

Reply:
f49cd887b7a3f356f581be3abfe8e5184a03d260 has the docs